modules/openy_gc_log/openy_gc_log.install

```diff
@@ -131,3 +131,28 @@ function openy_gc_log_update_8006() {
     $fields[$field_name]
   );
 }
+
+/**
+ * Import view config and update entity.
+ */
+function openy_gc_log_update_8007() {
+  $config_dir = drupal_get_path('module', 'openy_gc_log');
+  $config_dir .= '/config/install/';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'views.view.virtual_y_logs',
+  ]);
+
+  // Update entity to add the changed field.
```

Comment:
I think it is better to separate views update and field install.
Not required, but if you have time, better for reading.